linera-chain/src/data_types.rs

```diff
@@ -313,14 +313,14 @@ impl CertificateValue {
 
 /// A statement to be certified by the validators, with its hash.
 #[derive(Debug, PartialEq, Eq, Hash, Clone)]
-pub struct HashedValue {
+pub struct CertificateHashedValue {
```

Comment:
If we wanted to use the full name, it would have to be `HashedCertificateValue`.

Reply:
Separately, I also wonder if the change is useful.

Reply:
IMHO it will improve the readability of the code. I'm saying that because I read the code with limited context about what was going on, and the naming felt confusing to me. `HashedValue` with no context seems like a generic value hashed. Unless `Value` means something specific in our system, which is not the case AFAIK.
Happy to discuss more though :)

Reply:
It's just a bit long. How do you all feel about `CertValue` and `HashedCertValue`?

Reply:
In general I'm ok with abbreviations, as long as it's obvious what they're short for (like this case), but I think our naming conventions are generally against abbreviations https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#naming-conventions, with some exceptions. Up to you and @ma2bd if we want to make an exception for this :)

Reply:
Right; I'm okay with `HashedCertificateValue`, too.

Reply:
I can live with `*CertValue` but we'd have to check if if there are other `Certificate*` names to change to be consistent.

Reply:
Changed some names to be consistent, lmk what you guys think

Reply:
`LiteCert` is a bit weird. Can we just go for `HashedCertificateValue` and call it a day?

Reply:
Later we can always rename `CertificateValue` into something else (`Receipt`?)